src/components/Trade/Components/SilppagePopover.tsx

```diff
@@ -0,0 +1,166 @@
+import {
+  Alert,
+  AlertDescription,
+  AlertIcon,
+  Button,
+  ButtonGroup,
+  FormControl,
+  IconButton,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Popover,
+  PopoverBody,
+  PopoverContent,
+  PopoverTrigger,
+  useColorModeValue,
+} from '@chakra-ui/react'
+import { bnOrZero } from '@shapeshiftoss/chain-adapters'
+import { debounce } from 'lodash'
+import { useCallback, useMemo, useRef, useState } from 'react'
+import { FaSlidersH } from 'react-icons/fa'
+import { useTranslate } from 'react-polyglot'
+import { HelperTooltip } from 'components/HelperTooltip/HelperTooltip'
+import { Row } from 'components/Row/Row'
+import { Text } from 'components/Text'
+import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+
+enum SlippageType {
+  Auto = 'Auto',
+  Custom = 'Custom',
+}
+
+const defaultSlippage = '0.10'
+const maxSlippage = '50'
```

Comment:
this is concerningly high, maybe 3% max? our users will get rekt on this

Reply:
Indeed. Can we also specify units in the naming? It looks like this is percentage (other examples include bps, decimal percentage, etc).